command_line_rock_paper_scissors.py

```diff
@@ -31,7 +31,11 @@ def main():
     game = Game(human, computer)
 
     while game.rounds == 0:
-        game.rounds = input('How many odd rounds would you like to play? ')
+        try:
+            game.rounds = input('How many odd rounds would you like to play? ')
+        except Exception:
```

Comment:
As discussed, this should make use of `except ... as ...` syntax in order to allow reuse of the specific exception message defined in the `Game.rounds` setter instead of needing to print a more generic string on the following line.